03-transactions.md

```diff
@@ -822,292 +820,292 @@ And here are the test vectors themselves:
-    output htlc_timeout_tx 3: 0200000000010182fafbf148ac73a8a8cbbdf459408e1d368b4a2498bb0305a703cb800ae03b0e03000000000000000001f1090000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500483045022100815d4ea550897aac19da74f77e057ef0ff5640714a5a138943533111326b71650220136189c225d4981f68f53aa2815beb788ee4d072e8290cb99afa301a57f5f5d301483045022100a074832b8f1ab20a623fb4aa5e1173001f04c7b2cbcc9b87572c1c12a1bf3b1f02200a47b31262dbea329ae0f70b278c34b55176993d038f57a264e10b62b9d017aa01006921039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c820120876475527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae67a9148a486ff2e31d6158bf39e2608864d63fefd09d5b88ac68f7010000
-    # local_signature = 304502210082bb528c57eb739edd87307fdc55f6f22dc33ef16aecdf3b42d8ad5c6b6ee7f202207cd93b27fb31301abfb1ef2f6dc3b96c0e059ecd4433933129e849126ac216d5
-    output htlc_success_tx 4: 0200000000010182fafbf148ac73a8a8cbbdf459408e1d368b4a2498bb0305a703cb800ae03b0e04000000000000000001d90d0000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500483045022100d388ded6a2cb9ebd6480342bd48ba28543b77bea5ea08c4b4b4a11751a13ced7022060aea2991e0f8c45a588ec0bf1bfc8ea939ee45e12edadba103d6fb49e2f76be0148304502210082bb528c57eb739edd87307fdc55f6f22dc33ef16aecdf3b42d8ad5c6b6ee7f202207cd93b27fb31301abfb1ef2f6dc3b96c0e059ecd4433933129e849126ac216d5012004040404040404040404040404040404040404040404040404040404040404046e21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c8201208763a91418bc1a114ccf9c052d3d23e28d3b0a9d1227434288527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae677502f801b175ac6800000000
+    remote_htlc_signature = 3045022100a0d07c88f68bd9d3a721a159741b374c7a195393acded34344a9a1f4d42070630220228221c3076eb512f4115ae33fb0cd384b5fcd64dabc11d20d70dea18c6e7dd8
+    # local_signature = 30440220523a2c62ac50f4795aa01793eea6c510b20bc7051b41fd44c57219431c1a9cee022003a76a1e4a6f25a1834532f73d02cf125197f3a129bb40a273cba59aaf4e6535
+    output htlc_success_tx 0: 020000000001017644b6a9fb53bbc752147b6bb88b1a90ab5bfca44d7a87b8f993f83446b418b30000000000000000000121020000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500473044022003f201eea4aa28831c8a0552e791d81a3ff454e97bdc688f97a1635692992a3602203e4c9362eaa04b97b94cf7a44161b932073dfcf683b1cf34904fa266ae47b14f014730440220523a2c62ac50f4795aa01793eea6c510b20bc7051b41fd44c57219431c1a9cee022003a76a1e4a6f25a1834532f73d02cf125197f3a129bb40a273cba59aaf4e6535012000000000000000000000000000000000000000000000000000000000000000006e210394854aa6eab5b2a8122cc726e9dded053a2184d88256816826d6231c068d4a5b7c8201208763a914b8bcb07f6344b42ab04250c86a6e8b75d3fdbbc688527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae677502f401b175ac6800000000
```

Comment:
Ok, there is still one last doubt: it seems that you're using the HTLC success tx weight when computing the fee paid by HTLC timeout txs (sorry for not reporting this before).